src/pybind/mgr/pg_autoscaler/module.py

```diff
@@ -527,6 +527,13 @@ def _calc_final_pg_target(
         ))
         return final_ratio, pool_pg_target, final_pg_target
 
+    def get_dynamic_threshold(final_pg_num: int, default_threshold: float) -> float:
+        if final_pg_num in (512, 1024):
+            return 1.0
```

Comment:
1.1 here suggest?

Reply:
Let's wait for @kamoltat's feedback

Reply:
@anthonyeleven @pdvian Here is my thoughts: https://github.com/ceph/ceph/pull/65687#discussion_r2394827985